### PR TITLE
Add some phpdoc cs rules

### PR DESCRIPTION
**Pull request by JesusValera**
# 📚  Description

I added the following rules to `.php_cs.dist` file:

- _phpdoc_add_missing_param_annotation_ [(src)](https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:phpdoc_add_missing_param_annotation) -> Add some `mixed` arguments on functions.
- _phpdoc_annotation_without_dot_ [(src)](https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:phpdoc_annotation_without_dot)-> Remove the dot from the `@param` tags.
- _phpdoc_summary_ [(src)](https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:phpdoc_summary)-> Add a final dot on PHPdocs summaries.

The `phpdoc_annotation_without_dot` doesn't affect the PHP summaries, and the only available rule for this purpose is `phpdoc_summary` which add the final dot. So, I think it is the most suitable solution. 🙂 

# Changes

- Apply mentioned cs fix rules.
- Remove some useless `@params` and descriptions.
- Fix some incorrect `@param` names (`src/php/Destructure.php`).

### CODE REVIEW

File: src/php/GlobalEnvironment.php [hunk 1]
```diff
@@ -77,9 +77,9 @@ public function addRequireAlias(string $inNamespace, Symbol $name, Symbol $fullN
      * Checks if an require alias exists.
      *
      * @param string $inNamespace The namespace in which the alias should exist
-     * @param Symbol The alias name
+     * @param Symbol $name The alias name
      *
-     * @return boolean
+     * @return bool
```
Comment (Chemaclass) [inline src/php/GlobalEnvironment.php:82]: You don't need this line/phpdoc

File: src/php/Lang/Tuple.php [hunk 2]
```diff
@@ -46,7 +45,7 @@ public static function create(...$values): Tuple
     /**
      * Create a new bracket Tuple.
      *
-     * @param AbstractType|scalar|null ...$values The values
+     * @param AbstractType|scalar|null ...$values
```
Comment (Chemaclass) [inline src/php/Lang/Tuple.php:48]: I never thought about this `AbstractType|scalar|null ...$values`, using the `...`there. But I like it 👍🏼

**Review by Chemaclass — APPROVED** [review]

File: src/php/GlobalEnvironment.php (hunk 1 above)
Comment (JesusValera) [reply src/php/GlobalEnvironment.php]: upps. Deleted 👍🏼

**Review by jenshaase — APPROVED** [review]

**Comment by jenshaase** [comment]
Good job. I merged it!

**Pull request merged**